release-notes/opensearch-neural-search.release-notes-2.18.0.0.md

```diff
@@ -3,6 +3,9 @@
 
 Compatible with OpenSearch 2.18.0
 
+### Features
+- Introduces ByFieldRerankProcessor for second level reranking on documents ([#932](https://github.com/opensearch-project/neural-search/pull/932))
+
```

Comment:
Remove this extra line 8. In the final formatting it will add 2 lines.

Reply:
fixed it in main while merging PR https://github.com/opensearch-project/neural-search/pull/956